testsuite/integration/microprofile-tck/config/pom.xml

```diff
@@ -111,27 +111,38 @@
                             <goal>execute-commands</goal>
                         </goals>
                         <configuration>
+                            <offline>true</offline>
+                            <jboss-home>${basedir}/target/wildfly</jboss-home>
+                            <stdout>${project.build.directory}/wildfly/standalone/log/weld-configuration.log</stdout>
+                            <java-opts>${modular.jdk.args}</java-opts>
+                            <system-properties>
+                                <maven.repo.local>${maven.repo.local}</maven.repo.local>
```

Comment:
Is there a reason this is defined here and not in the `<java-opts>`? It seems we should be consistent if they achieve the same thing.

Reply:
@jamezp This is the `execute-commands` goal. System-properties maven-repo-local is already set that way (I just moved the `<configuration>` element.
 The `package` goal has a different `<packaging-scripts>` element that defines the `<java-opts>`.